README.md

```diff
@@ -201,6 +201,21 @@ choco install etcher
 ```sh
 choco uninstall etcher
 ```
+#### Scoop (Windows (PowerShell))
+
+This package is maintained by Scoop Extras Bucket, and is kept up to date automatically.
+
+```sh
+scoop bucket add extras
+scoop update (Updates buckets) | scoop update etcher (Updates Balena Etcher) | scoop update * (Updates everything)
```

Comment:
Does the user have to type in all of this line?

Reply:
No, it is enough to install with one command, and you can update via the * flag (update all), or by the application flag (only it will be updated)